desktop/flipper-common/src/settings.tsx

```diff
@@ -90,6 +90,8 @@ export type ProcessConfig = {
   launcherMsg: string | null;
   // Controls whether to delegate to the launcher if present.
   launcherEnabled: boolean;
+  // Control whether to enable "update available" notifications
+  notifyAvailableUpdate: boolean;
```

Comment:
Would you mind negating the flag, so that the behavior is more intuitive if not set, in which case it is the same as `false`?. E.g. `suppressPluginUpdateNotifications`?

Reply:
@mweststrate changes made...anything else I should change?